engine/classes/ElggPlugin.php

```diff
@@ -260,7 +260,7 @@ public function normalizePriority($priority) {
 		// if no priority assume a priority of 1
 		$old_priority = $this->getPriority();
 		$old_priority = $old_priority ? : 1;
-		$max_priority = _elgg_get_max_plugin_priority();
+		$max_priority = _elgg_get_max_plugin_priority() ? : 1;
```

Comment:
can `_elgg_get_max_plugin_priority` not return 1?

Reply:
It can return 0